src/app/work-item/work-item-list/work-item-list.component.scss

```diff
@@ -28,5 +29,29 @@
     @include transition(opacity .3s);
     opacity: .6;
     width: 100%;
-  }
+  }  
+}
+
+.detail-panel {
+  background: #ffffff;  
+  position: fixed;
+  left: auto;
+  top: 35px;
```

Comment:
@mindreeper2420 Yes, so I did all these crap, I'll sit with @SMahil once she is back in office on Monday and update al the CSS work. Excuse my less knowledge on CSS for now!!